eng/pipelines/runtime.yml

```diff
@@ -271,6 +271,47 @@ jobs:
           eq(variables['monoContainsChange'], true),
           eq(variables['isFullMatrix'], true))
 
+#
+# Build the whole product using Mono and run libraries tests, multi-scenario
+#
+- template: /eng/pipelines/common/platform-matrix.yml
```

Comment:
I would suggest adding this to runtime-staging once this is ready to be merged (I know we're on early process gathering info on how many tests fail, etc) to get it stable.

Reply:
The lane passed after running just `System.Buffers.Tests.csproj` :) 
https://dev.azure.com/dnceng/public/_build/results?buildId=948229&view=logs&j=f217f715-90ad-5404-53bd-b93e3b80f465&t=9df76bd4-3041-5537-002b-9e29a53263b9
Moved the block to `runtime-staging.yml`